components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -254,16 +254,23 @@ DomainBlockingType GetDomainBlockingType(HostContentSettingsMap* map,
   if (!brave_shields::GetBraveShieldsEnabled(map, url))
     return DomainBlockingType::kNone;
 
-  // Block if ad blocking is "aggressive"
-  if (brave_shields::GetCosmeticFilteringControlType(map, url) ==
-      ControlType::BLOCK) {
+  // Don't block if ad blocking is off.
+  if (brave_shields::GetAdControlType(map, url) != ControlType::BLOCK)
+    return DomainBlockingType::kNone;
+
+  const ControlType cosmetic_control_type =
+      brave_shields::GetCosmeticFilteringControlType(map, url);
+  // Block if ad blocking is "aggressive".
+  if (cosmetic_control_type == ControlType::BLOCK) {
     return DomainBlockingType::kAggressive;
   }
 
-  if (base::FeatureList::IsEnabled(
+  // Block using 1PES if ad blocking is "standard".
+  if (cosmetic_control_type == BLOCK_THIRD_PARTY &&
```

Comment:
not sure I understand why it is about cosmetic filters, but that's not relevant to the PR...

Reply:
this is definitely a bit confusing and we need to consolidate the logic for adblock with its own control type. RIght now we're duplicating checks in `BraveShieldsDataController` and presumably in android for the UI setting

Reply:
https://github.com/brave/brave-browser/issues/39197